objects/src/notes/nullifier.rs

```diff
@@ -60,6 +62,12 @@ impl Nullifier {
     }
 }
 
+impl Display for Nullifier {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        Display::fmt(&self.0, f)
```

Comment:
I think we want to output the hex string here, right? If so, I'd probably write this as:
```Rust
write!(f, "{}", self.to_hex())
```
Also, should we override `Debug` implementation to work in the same way too?

Reply:
> I think we want to output the hex string here, right? If so, I'd probably write this as:

We actually do the same, because `Display` for `RpoDigest` is implemented as `self.to_hex()`.

> Also, should we override Debug implementation to work in the same way too?

Yes, I think, it would be nice.

Reply:
> We actually do the same, because `Display` for `RpoDigest` is implemented as `self.to_hex()`.

I'd still make it explicit here in case the implementation of `RpoDigest::fmt` changes at some point down the road.